js/iframe-coordinator/elements/frame-router.ts

```diff
@@ -1,13 +1,30 @@
 import { Host } from "../elm/Host.elm";
 
+const TOAST_REQUEST_MSG_TYPE = 'toastRequest';
+const TOAST_REQUEST_EVENT_TYPE = 'toastRequest';
```

Comment:
Should we export this constant so it's available for users?  The event and type is documented below.